rel-eng/openshift-sdn-node.service

```diff
@@ -3,8 +3,7 @@ Description=OpenShift SDN Node
 After=openvswitch.service
 After=openshift-sdn-master.service
 After=openshift-master.service
-Wants=openshit-sdn-master.service
-Wants=openshift-master.service
+After=etcd.service
```

Comment:
It's my understanding that openshift-sdn-node still talks to etcd rather than the API. If etcd is installed and enabled on the same host as openshift-sdn-node we should start after it's been started.